yt_dlp/extractor/goodgame.py

```diff
@@ -30,28 +30,30 @@ class GoodGameIE(InfoExtractor):
 
     def _real_extract(self, url):
         channel_name = self._match_id(url)
-        response = self._download_json(f'https://api2.goodgame.ru/v2/streams/{channel_name}', channel_name)
-        player_id = response['channel']['gg_player_src']
+        response = self._download_json(f'https://goodgame.ru/api/4/users/{channel_name}/stream', channel_name)
+        player_id = response['streamkey']
 
         formats, subtitles = [], {}
-        if response.get('status') == 'Live':
+        if response.get('status'):
             formats, subtitles = self._extract_m3u8_formats_and_subtitles(
                 f'https://hls.goodgame.ru/manifest/{player_id}_master.m3u8',
                 channel_name, 'mp4', live=True)
         else:
             self.raise_no_formats('User is offline', expected=True, video_id=channel_name)
 
+        description_page = self._download_webpage(f'https://goodgame.ru/ajax/channel/tabs?id={player_id}', player_id)
+        description = '\n'.join(description_page.splitlines()[21:-47])
```

Comment:
Imo. the extra request for the description is not worth it. Arguably, this is also more the channel's about page than an actual description.
```suggestion
```